google/cloud/security/inventory/pipelines/load_projects_cloudsql_pipeline.py

```diff
@@ -0,0 +1,358 @@
+                            formatted_expirationtime = (
+                                parsed_time\
+                                    .strftime(self.MYSQL_DATETIME_FORMAT))
+                        except (TypeError, ValueError, AttributeError) as e:
+                            LOGGER.error(
+                                'Unable to parse timeCreated' +\
+                                'from authorizednetworks: %s\n%s',
+                                network.get('expirationTime', ''), e)
+                            formatted_expirationtime = '1972-01-01 00:00:00'
+                    else:
+                        formatted_expirationtime = '1972-01-01 00:00:00'
+
+                    yield {
+                        'project_number': instances_map['project_number'],
+                        'instance_name': item.get('name'),
+                        'kind': network.get('kind'),
+                        'name': network.get('name'),
+                        'value': network.get('value'),
+                        'expirationTime': formatted_expirationtime
+                    }
+
+    def _transform_ipaddresses(self, cloudsql_instances_map):
+        """Yield an iterator of loadable ipAddresses of cloudsql instances.
+
+        Args:
+            cloudsql_instnces_maps: An iterable of instances as per-project
+                dictionary.
+                Example: {'project_number': 11111,
+                          'instances': instances_dict}
+
+        Yields:
+            An iterable of ipAddresses dictionary.
+        """
+
+        for instances_map in cloudsql_instances_map:
+            instances = instances_map['instances']
+            items = instances.get('items', [])
+            for item in items:
+                ipaddresses = item.get('ipAddresses', [{}])
+                for ipaddress in ipaddresses:
+                    if ipaddress.get('timeToRetire') is not None:
+                        try:
+                            parsed_time = dateutil_parser\
+                                .parse(ipaddress.get('timeToRetire'))
+                            formatted_timetoretire = (
+                                parsed_time\
+                                    .strftime(self.MYSQL_DATETIME_FORMAT))
+                        except (TypeError, ValueError, AttributeError) as e:
+                            LOGGER.error(
+                                'Unable to parse timeCreated' +\
+                                ' from ipaddresses: %s\n%s',
+                                ipaddress.get('timeToRetire', ''), e)
+                            formatted_timetoretire = '1972-01-01 00:00:00'
+                    else:
+                        formatted_timetoretire = '1972-01-01 00:00:00'
+
+                    yield {
+                        'project_number': instances_map['project_number'],
+                        'instance_name': item.get('name'),
+                        'ipAddress': ipaddress.get('ipAddress'),
```

Comment:
hope this doesn't throw things off, but for consistency, could you change this to `ip_address` instead of `ipAddress`?